test/api_utils_test.dart

```diff
@@ -62,9 +63,9 @@ void main() {
   test('post', () async {
     var newPost = Post(title: 'title', body: 'body');
     var response = await post(
-      url: 'https://jsonplaceholder.typicode.com/posts',
-      body: newPost.toJson(),
-    );
+        url: 'https://jsonplaceholder.typicode.com/posts',
+        body: newPost.toJson(),
+        headers: {});
```

Comment:
With the changes we made to headers, we shouldn’t need to pass empty maps in the tests